package.json

```diff
@@ -32,24 +32,25 @@
     "@babel/plugin-transform-modules-commonjs": "^7.12.1",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-typescript": "^7.12.7",
-    "@ceramicnetwork/cli": "^1.0.0-rc.13",
-    "@ceramicnetwork/common": "^1.0.0-rc.6",
-    "@ceramicnetwork/core": "^1.0.0-rc.12",
-    "@ceramicnetwork/streamid": "^1.0.0-rc.3",
-    "@ceramicnetwork/stream-tile": "^1.0.0-rc.6",
-    "@ceramicnetwork/http-client": "^1.0.0-rc.6",
-    "@ceramicnetwork/ipfs-daemon": "^1.0.0-rc.9",
+    "@ceramicnetwork/cli": "^1.1.0-rc",
+    "@ceramicnetwork/common": "^1.1.0-rc",
+    "@ceramicnetwork/core": "^1.2.0-rc",
+    "@ceramicnetwork/http-client": "^1.0.7-rc",
+    "@ceramicnetwork/ipfs-daemon": "^1.0.7-rc",
+    "@ceramicnetwork/stream-tile": "^1.0.7-rc",
+    "@ceramicnetwork/streamid": "^1.0.0-rc",
     "@jest/reporters": "^26.6.2",
     "@types/jest": "^26.0.19",
     "babel-jest": "^26.6.3",
+    "blockcodec-to-ipld-format": "^1.0.0",
     "dag-jose": "^0.3.0",
     "dids": "^2.1.0",
-    "ipfs-http-client": "~49.0.4",
+    "ipfs-http-client": "^50.1.2",
     "jest": "^26.6.3",
     "jest-resolver-enhanced": "^1.0.0",
     "key-did-provider-ed25519": "^1.1.0",
     "key-did-resolver": "^1.1.2-rc2",
-    "multiformats": "3.0.3",
+    "multiformats": "^4.6.3",
```

Comment:
Is `multiformats` really needed now that we have `blockcodec-to-ipld-format`?

Reply:
oh good point. will remove it